src/main/webapp/WEB-INF/tags/instructor/results/questionPanel.tag

```diff
@@ -25,7 +25,7 @@
                 <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYSECTION%>" value="${data.selectedSection}">
             </form>
             <div class='display-icon pull-right'>
-                <span class="glyphicon ${ isShowingResponses ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down'} pull-right"></span>
```

Comment:
hi @AnaghHegde , I see that you changed the default value of to both glyphicon-cheron-up. However, I do not think this is the best approach. If possible, could you go deeper and find out what is isShowingResponses doing and why glyphicon-chevron-up is chosen instead of glyphicon-chevron-down. If there is a logical error, please update the page data as well.